### ISSUE TITLE
Safari Screen Reader Accessibility Issue

### ISSUE THREAD

**Issue by alexandriacuellar** [dhg/Skeleton#142]
For me it is important that the Safari Screen Reader is able to read all the text on my webpage pertinent to an article. It unfortunately was ignoring list data that I was displaying in columns. After a lot of testing I discovered that the issue was the css float property on the column and columns class. 

I edited skeleton.css to use display:inline-block and vertical-align:top in place of float:left and now the missing data appears in the Safari Screen Reader. 

The code looks like this now: 

.container .column,
    .container .columns                         { display: inline-block; margin-left: 10px; margin-right: 10px; vertical-align:top;}

I haven't tested cross-browser compatibility on it but it should work on all modern browsers and if it works well with the Safari Screen Reader I think it will work with others.

**Comment by dhg** [comment]
This is an interesting issue. Using `floats` for content is extremely common, so it surprises me that Safari Screen Reader skips over it, but I have to admit I don't have a ton of experience using screen readers. I'm going to leave this issue open as Skeleton still uses `floats`. It may take a bit, but I'm going to try and test with screen readers and see if Skeleton falls down. Thanks for letting me know.
